src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Filter/DateTimeFilterIntegration.php

```diff
@@ -12,90 +11,129 @@
 
-        $result = $this->execute([['updated', Operators::NOT_EQUAL, '2016-08-29 12:00:00']]);
+        $result = $this->execute([['updated', Operators::NOT_EQUAL, $updatedAt]]);
+        $this->assert($result, ['foo', 'baz']);
+
+        $currentDate = new \DateTime('now', new \DateTimeZone('UTC'));
+        $result = $this->execute([['updated', Operators::NOT_EQUAL, $currentDate->format('Y-m-d H:i:s')]]);
         $this->assert($result, ['bar', 'baz', 'foo']);
     }
 
     public function testOperatorBetween()
     {
-        $result = $this->execute([['updated', Operators::BETWEEN, ['2016-08-04 01:28:51', '2016-08-25 00:00:00']]]);
+        $result = $this->execute([['updated', Operators::BETWEEN, [static::$createdDates['before_second'], static::$createdDates['after_all']]]]);
         $this->assert($result, ['bar', 'baz']);
 
-        $result = $this->execute([['updated', Operators::BETWEEN, ['2016-08-04 01:28:51', '2016-08-24 23:59:59']]]);
+        $result = $this->execute([['updated', Operators::BETWEEN, [static::$createdDates['before_second'], static::$createdDates['before_third']]]]);
         $this->assert($result, ['bar']);
 
-        $result = $this->execute([['updated', Operators::BETWEEN, ['2016-08-29 00:00:01', '2016-08-29 00:00:01']]]);
+        $currentDate = new \DateTime('now', new \DateTimeZone('UTC'));
+        $result = $this->execute([['updated', Operators::BETWEEN, [static::$createdDates['after_all'], $currentDate]]]);
         $this->assert($result, []);
     }
 
     public function testOperatorNotBetween()
     {
-        $result = $this->execute([['updated', Operators::NOT_BETWEEN, ['2016-08-04 01:28:51', '2016-08-25 00:00:00']]]);
+        $result = $this->execute([['updated', Operators::NOT_BETWEEN, [static::$createdDates['before_second'], static::$createdDates['after_all']]]]);
         $this->assert($result, ['foo']);
 
-        $result = $this->execute([['updated', Operators::NOT_BETWEEN, ['2016-08-04 01:28:51', '2016-08-24 23:59:59']]]);
+        $result = $this->execute([['updated', Operators::NOT_BETWEEN, [static::$createdDates['before_second'], static::$createdDates['before_third']]]]);
         $this->assert($result, ['baz', 'foo']);
 
-        $result = $this->execute([['updated', Operators::NOT_BETWEEN, ['2016-08-29 00:00:01', '2016-08-29 00:00:01']]]);
+        $currentDate = new \DateTime('now', new \DateTimeZone('UTC'));
+        $result = $this->execute([['updated', Operators::NOT_BETWEEN, [static::$createdDates['after_all'], $currentDate]]]);
         $this->assert($result, ['bar', 'baz', 'foo']);
     }
 
     /**
      * @expectedException \Akeneo\Component\StorageUtils\Exception\InvalidPropertyTypeException
-     * @expectedExceptionMessage Property "updated" expects an array with valid data, should contain 2 strings with the format "yyyy-mm-dd H:i:s".
+     * @expectedExceptionMessage Property "updated" expects an array with valid data, should contain 2 strings with the format "Y-m-d H:i:s".
```

Comment:
not sure we can change the format to 'Y-m-d', for a non tech user, 'Y' means nothing.